simulator/interface/gdb/gdb_interface.cpp

```diff
@@ -0,0 +1,184 @@
+/**
+ * gdb_interface.c - Functional simulator interfaces for GDB
+ * @author Vyacheslav Kompan
+ * Copyright 2018 MIPT-MIPS
+ */
+
+
+/* BFD config */
+#include <bfd/config.h>
+/* GDB Interface */
+#include <gdb/remote-sim.h>
+#include <gdb/callback.h>
+/* GDB Simulator utilities */
+#include <sim-config.h>
+#include <sim-types.h>
+#include <sim-inline.h>
+#include <sim-arange.h>
+#include <sim-base.h>
+#include "sim-main.h"
+/* MIPT-MIPS simulator interfaces */
+#include <simulator.h>
+#include <infra/config/config.h>
+#include <mips/mips.h>
+#include <func_sim/func_sim.h>
```

Comment:
Same for that.

Reply:
This header contains declaration of `BearingLost`